### PR TITLE
docs: document how to use tab blocks

### PR DESCRIPTION
**Pull request by fhennig**
This is an extension feature, I wanted to explicitly document it in the contribution guide

### CODE REVIEW

**Comment by fhennig** [comment]
The new page: https://deploy-preview-649--stackable-docs.netlify.app/home/nightly/contributor/docs/using-tab-blocks

File: modules/contributor/pages/docs/using-tab-blocks.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+= Using tab blocks
+:asciidoctor-tabs-gh: https://github.com/asciidoctor/asciidoctor-tabs
+
+The {asciidoctor-tabs-gh}[`asciidoctor/tabs`] extension is installed, so you can use tab blocks.
+An example can be found in the xref:management:stackablectl:installation.adoc[stackablectl installation instructions].
+
```
Comment (NickLarsenNZ) [inline modules/contributor/pages/docs/using-tab-blocks.adoc:6]:
```suggestion

TIP: Be sure to use different block indicators for tab block and admonition blocks.

```

**Review by NickLarsenNZ — APPROVED** [review]
LGTM, added a suggestion

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

**Pull request merged**